pysiaf/tests/test_rotations.py

```diff
@@ -0,0 +1,102 @@
+import numpy as np
+from math import sin, cos, acos, pi
+from pysiaf.utils import rotations as rt
+import pylab as pl
+
+# Some values used in both tests
+#  Set up some arbitrary values
+ra = 20.0
+dec = 70.0
+roll = 15.0
+v2 = 200.0
+v3 = 300.0
+ra2 = 21.0
+dec2 = 69.0
+
+
+def test_attitude(verbose=False):
+    """ Tests the properties of the attitude matrix in calculating positions and roll angles. Incidentally tests underlying
+    matrix generation and vector construction  and conversions to Euler angles
+
+    Parameters
+    ----------
+    verbose     logical - set to true if detailed print-out needed
```

Comment:
Please adhere to numpy docstring formatting, i.e. add the argument type after `:` and move the description to a new line. See https://numpydoc.readthedocs.io/en/latest/format.html#documenting-classes

Reply:
I have attended to this